backend/cnn/upload_data.py

```diff
@@ -0,0 +1,60 @@
+from azure.storage.blob import BlobServiceClient, BlobClient, ContainerClient
+from azure.identity import DefaultAzureCredential
+import pandas as pd
+import os
+
+# Set your storage account details
+account_name = ''
+account_key = ''
+container_name = ''
+local_directory_path = os.path.join(os.getenv("DATASET_PATH"), 'TRAIN_DATASET')
```

Comment:
We could add a way to load this from an .env file, but currently it's not a priority.